.changeset/fuzzy-papayas-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'doctocat-nextjs-site': patch
```

Comment:
This should be a minor, but recommend merging with the other changeset instead which is already a minor.